### PR TITLE
CFM-1239: Create CTA buttons to initiate new group creation

### PR DESCRIPTION
**Pull request by DavidHernandez**
Adds the CTA block to the homepage and adds an action to add a button to create a group.

Issue: https://issuetracker.amplexor.com/jira/browse/CFM-1239

cta-add-group

### CODE REVIEW

File: project/profiles/capacity4more/modules/c4m/features/c4m_features_homepage/c4m_features_homepage.features.inc [hunk 1]
```diff
@@ -1,5 +1,4 @@
 <?php
-
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/features/c4m_features_homepage/c4m_features_homepage.features.inc]: Revert this change

File: project/profiles/capacity4more/modules/c4m/features/c4m_features_homepage/c4m_features_homepage.pages_default.inc [hunk 2]
```diff
@@ -1,5 +1,4 @@
 <?php
-
```
Comment (ordavidil) [inline project/profiles/capacity4more/modules/c4m/features/c4m_features_homepage/c4m_features_homepage.pages_default.inc]: revert this change

**Comment by jover** [comment]
This PR was merged into `feature/CFM-1205-cta-tasks-hook`, but this one was already merged into `develop` so I did this manually.
